### PR TITLE
Improved CLI Readability

### PR DESCRIPTION
**Pull request by Zatine**
Made column sizes more appropriate when listing workflows using CLI, and also truncating backfill descriptions by default. To not truncate you supply the command "--no-trunc" similar to Docker, and then you will get the full descriptions.

### CODE REVIEW

File: styx-cli/src/test/java/com/spotify/styx/cli/PrettyCliOutputTest.java [hunk 1]
```diff
@@ -37,16 +37,25 @@
...
-      .id("backfill-2")
-      .start(Instant.parse("2017-01-01T00:00:00Z"))
-      .end(Instant.parse("2017-01-02T00:00:00Z"))
-      .workflowId(WorkflowId.create("component", "workflow2"))
-      .concurrency(2)
-      .description("Description")
-      .nextTrigger(Instant.parse("2017-01-01T00:00:00Z"))
-      .schedule(Schedule.DAYS)
-      .build();
+  private Backfill backfill(String description) {
```
Comment (honnix) [inline styx-cli/src/test/java/com/spotify/styx/cli/PrettyCliOutputTest.java]: Let’s move this method down after those properties and make it static.

**Comment by danielnorberg** [comment]
@honnix I think codecov is correctly indicating that changes are not covered by tests...

**Comment by honnix** [comment]
Hmm, I think we misunderstood it. We checked those red blocks in overview page and that’s why it is weird. Should instead check the diff tab. Will fix.

**Comment by honnix** [comment]
Let's add test cases to cover all the changes.

**Comment by honnix** [comment]
👍 awesome!

**Pull request merged**